### ISSUE TITLE
Show Feed Link global setting does not work

### ISSUE THREAD

**Issue by Feldon** [joomla/joomla-cms#7640]
#### Steps to reproduce the issue
In Article Options, set Integration --> Show Feed Link to "Hide". 
In Menu Item, set Integration --> Show Feed Link to "Use Global".

#### Expected result
The RSS feed option should be disabled for the page.

#### Actual result
The RSS feed option is still enabled for the page.

#### System information (as much as possible)
PHP Version 5.3.29
Web Server Apache/2.2.27 (Unix) mod_hive/5.0 mod_ssl/2.2.27 OpenSSL/1.0.1e-fips DAV/2 mod_bwlimited/1.4
Joomla! Version Joomla! 3.4.3 Stable [ Ember ] 2-July-2015 16:00 GMT
Joomla! Platform Version Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent Mozilla/5.0 (Windows NT 6.1; WOW64; rv:39.0) Gecko/20100101 Firefox/39.0 

#### Additional comments
Manually setting each Menu Item to "Hide" works as expected.  Its only the global Article Options that is not working.

**Comment by Bakual** [comment]
This depends on what type of menu item it is. If the menu item is pointing directly to the article in question, the article settings will be overruled by the menu item, thus using the global seeting.
If the menu item is eg a category list, then the article settings will have priority over the menu item.

Can you double check that?

**Comment by Feldon** [comment]
In the general "Article Options", which apply to all articles, there is a "Show Feed Link" option.  This is set to "Hide".
The menu item is for a "Category Blog".  It has its own "Show Feed Link" option set to "Use Global".

Thus, I would either expect the articles to respect their "Hide" setting OR I would expect the page to respect the menu item setting of "Use Global", which points to "Hide".  But instead the feed links are still being shown.  I have to further set the menu item option for "Show Feed Link" to "Hide" before the feed links are actually hidden.

In other words, the "Use Global" option serves no purpose.


Date: Wed, 5 Aug 2015 23:05:35 -0700

**Comment by Bakual** [comment]
Can't confirm. Disabling feed in global content options doesn't show the feed if the param is set to "Use Global" in the category blog or list view.
When you test, make sure you clear the caches (also in Joomla). It may be a cached page you see.

**Comment by brianteeman** [comment]
It has been several months since the last comment - closing<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7640">issues.joomla.org/joomla-cms/7640</a>.</sub>
